addons/account/account.py

```diff
@@ -23,14 +23,13 @@ class res_company(models.Model):
         string="Loss Exchange Rate Account", domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)])
 
 
-class account_payment_term(models.Model):
+class AccountPaymentTerm(models.Model):
     _name = "account.payment.term"
     _description = "Payment Term"
     _order = "name"
 
     name = fields.Char(string='Payment Term', translate=True, required=True)
-    active = fields.Boolean(string='Active', default=True,
-        help="If the active field is set to False, it will allow you to hide the payment term without removing it.")
+    active = fields.Boolean(string='Active', default=True, help="If the active field is set to False, it will allow you to hide the payment term without removing it.")
```

Comment:
You can remove the string parameter here, as the ORM will generate a capitalized version of the field name by default.